collector/parameters.go

```diff
@@ -0,0 +1,49 @@
+package collector
+
+import (
+	"github.com/globalsign/mgo"
+	"github.com/globalsign/mgo/bson"
+
+	"github.com/golang/glog"
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+var (
+	cursorTimeoutMillis = prometheus.NewGauge(prometheus.GaugeOpts{
+		Namespace: Namespace,
+		Subsystem: "parameters",
+		Name:      "cursor_timeout_millis",
+		Help:      "An integer that represents the cursorTimoutMillis option in mongod",
+	})
+	metric_mapping = map[string]prometheus.Gauge{
+		"cursorTimeoutMillis": cursorTimeoutMillis,
```

Comment:
can we make this a CLI flag to pass down?

Reply:
yes, i need to make logic changes since parameters can be bools too

Reply:
great, it'd be more useful to have as a config-driven option to allow people to (1) pick what options they want and (2) exclude the ones they don't